### PR TITLE
181 get rid of hamcrest lib

### PR DESCRIPTION
**Pull request by LEQADA**
## Description

Currently we are using 2 asserting libraries Hamcrest and AssertJ. Hamcrest should've been replaced with AssertJ. 

Resolves https://github.com/commercetools/commercetools-payone-integration/issues/181

### CODE REVIEW

File: service/src/test/java/com/commercetools/pspadapter/payone/transaction/TransactionBaseExecutorTest.java [hunk 1]
```diff
@@ -22,19 +21,19 @@ public void responseToJsonString() throws Exception {
                 "errorcode", "00123"
         ));
 
-        assertThat(jsonString, is(not(isEmptyOrNullString())));
+        assertThat(jsonString).isNotEmpty();
```
Comment (andrii-kovalenko-ct) [inline service/src/test/java/com/commercetools/pspadapter/payone/transaction/TransactionBaseExecutorTest.java]: `isNotBlanck()` even better
same on line [49](https://github.com/commercetools/commercetools-payone-integration/pull/184/files#diff-d7a6d71c83bdd9a5753a0e9241a935a4R49)

File: service/src/test/java/com/commercetools/pspadapter/payone/transaction/TransactionBaseExecutorTest.java [hunk 2]
```diff
@@ -47,16 +46,15 @@ public void exceptionToResponseJsonString() throws Exception {
...
-        assertThat(node.has(STATUS), is(true));
-        assertThat(node.has(ERROR_CODE), is(true));
-        assertThat(node.has(CUSTOMER_MESSAGE), is(true));
+        assertThat(node.has(STATUS)).isTrue();
+        assertThat(node.has(ERROR_CODE)).isTrue();
+        assertThat(node.has(CUSTOMER_MESSAGE)).isTrue();
 
-        assertThat("Error message node should contain exception message",
-                node.get(ERROR_MESSAGE).asText(), containsString(TEST_ERROR_MESSAGE));
+        assertThat(node.get(ERROR_MESSAGE).asText()).as("Error message node should contain exception message").contains(TEST_ERROR_MESSAGE);
```
Comment (andrii-kovalenko-ct) [inline service/src/test/java/com/commercetools/pspadapter/payone/transaction/TransactionBaseExecutorTest.java]: we don't have hard line length limit but we strongly encourage to split long lines

Comment (LEQADA) [reply service/src/test/java/com/commercetools/pspadapter/payone/transaction/TransactionBaseExecutorTest.java]: https://github.com/commercetools/commercetools-payone-integration/pull/184/commits/111a893c304719b668c15c0d09a2c63a8a891abe#diff-d7a6d71c83bdd9a5753a0e9241a935a4R57

File: service/src/test/java/com/commercetools/pspadapter/payone/transaction/TransactionBaseExecutorTest.java (hunk 1 above)
Comment (LEQADA) [reply service/src/test/java/com/commercetools/pspadapter/payone/transaction/TransactionBaseExecutorTest.java]: https://github.com/commercetools/commercetools-payone-integration/pull/184/commits/111a893c304719b668c15c0d09a2c63a8a891abe#diff-d7a6d71c83bdd9a5753a0e9241a935a4R24
https://github.com/commercetools/commercetools-payone-integration/pull/184/commits/111a893c304719b668c15c0d09a2c63a8a891abe#diff-d7a6d71c83bdd9a5753a0e9241a935a4R49

**Comment by andrii-kovalenko-ct** [comment]
closes #182

**Pull request merged**